fieldservice_web_timeline_view/static/src/js/fsm_gantt.js

```diff
@@ -0,0 +1,221 @@
+odoo.define('fieldservice_web_timeline_view.fsm_gantt', function (require) {
```

Comment:
Why is this JS here? Isn't the Timeline feature already provided by the Web Timeline dependency module?

Reply:
Web Timeline module JS is extended here because of display FSM Person and Stage wise color (Order) in Timeline View.

![web_timeline](https://user-images.githubusercontent.com/20418904/76764898-9be14b00-67bb-11ea-9bd1-05ac6f2a301c.png)

Reply:
AFAIK, you can already put what you want on each timeline item through QWeb templates. See https://github.com/OCA/project/blob/12.0/project_timeline_hr_timesheet/views/project_task_view.xml

Reply:
@pedrobaeza , @dreispt 

We have done customization on web_timeline view for extra field-service features:-
  1) Added a custom filter functionality.
  2) Customize color functionality with a stage, in a stage, In an fsm.stage has choose_color field, and this field color display in an FSM order record timeline view.
  3) All-time line view line display in a top bar.
  4) Display All FSM workers in a timeline view so easily assign an order.